### PR TITLE
Added the eyal accounts

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [x] documentation
- [ ] dependencies

## Description
Eyal accounts to cloud-governance jenkins

### CODE REVIEW

File: jenkins/tenant/aws/qe/qe_02/PolicyJenkinsfileDaily [hunk 1]
```diff
@@ -0,0 +1,91 @@
+accounts_list = ['konflux-fedora': 'dbaez@redhat.com, eedri@redhat.com, ilario@redhat.com, xjiang@redhat.com, ebattat@redhat.com, maniskum@redhat.com',
```
Comment (ebattat) [inline jenkins/tenant/aws/qe/qe_02/PolicyJenkinsfileDaily:1]: Why is it a list and not dictionary of list ?
{'konflux-fedora': ['dbaez@redhat.com, eedri@redhat.com, ilario@redhat.com, xjiang@redhat.com, ebattat@redhat.com, maniskum@redhat.com']}

**Review by ebattat — CHANGES_REQUESTED** [review]

File: jenkins/tenant/aws/qe/qe_02/PolicyJenkinsfileDaily (hunk 1 above)
Comment (athiruma) [reply jenkins/tenant/aws/qe/qe_02/PolicyJenkinsfileDaily:1]: jenkins supports dict like this.

Comment (ebattat) [reply jenkins/tenant/aws/qe/qe_02/PolicyJenkinsfileDaily:1]: Ok got u

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**